data/json/items/armor/suits_protection.json

```
@@ -830,5 +830,168 @@
     "valid_mods": [ "steel_padded", "alloy_padded" ],
     "environmental_protection": 5,
     "flags": [ "VARSIZE", "WATERPROOF", "POCKETS", "HOOD", "RAINPROOF", "STURDY" ]
+  },
+  {
+    "id": "exosuit_survivor",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "survivor power armor" },
+    "description": "Although power armor took the world's militaries by storm, not everyone has access to it.  This particular suit is cobbled together from scavenged materials, but it works for what it is.",
+    "weight": "34 kg",
+    "volume": "34 L",
+    "price_postapoc": "100 USD",
+    "material": [ "steel" ],
+    "symbol": "[",
+    "looks_like": "survivor_suit",
+    "color": "light_gray",
+    "covers": [ "torso", "arms", "hands", "legs", "feet" ],
+    "coverage": 95,
+    "encumbrance": 40,
+    "warmth": 20,
+    "material_thickness": 7,
+    "environmental_protection": 10,
+    "max_charges": 500,
+    "relic_data": {
+      "passive_effects": [
+        {
+          "has": "WORN",
+          "condition": "ACTIVE",
+          "values": [ { "value": "STRENGTH", "add": 2 }, { "value": "SPEED", "add": 15 } ]
+        }
+      ]
+    },
+    "use_action": {
+      "type": "transform",
+      "msg": "The %s engages.",
+      "target": "exosuit_survivor_on",
+      "active": true,
+      "need_worn": true,
+      "need_charges": 1
+    },
+    "flags": [ "USE_UPS", "NAT_UPS", "STURDY", "NO_UNLOAD", "NO_RELOAD", "OUTER" ]
+  },
+  {
+    "id": "exosuit_survivor_on",
+    "copy-from": "exosuit_survivor",
+    "repairs_like": "exosuit_survivor",
+    "type": "TOOL_ARMOR",
+    "name": { "str_sp": "survivor power armor (active)" },
+    "description": "Although power armor took the world's militaries by storm, not everyone has access to it.  This particular suit is cobbled together from scavenged materials, but it works for what it is.  It is currently turned on, activate it to turn it off.",
+    "extend": { "flags": [ "HEAVY_WEAPON_SUPPORT", "TRADER_AVOID" ] },
+    "power_draw": 500000,
+    "revert_to": "exosuit_survivor",
+    "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "The %s disengages.", "target": "exosuit_survivor" },
+    "encumbrance": 10,
+    "environmental_protection": 16,
+    "weight_capacity_bonus": "34 kg"
+  },
+  {
+    "id": "exosuit_military",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "military exo-suit" },
+    "description": "Although power armor took the world's militaries by storm, the technologies used to make them also allowed for smaller, less exclusive external rigging that rely on UPS to boost strength, ones designed for operatives or light infantry.",
```

```suggestion
    "description": "Although power armor took the world's militaries by storm, the technologies used to make them also allowed for smaller, less protective external rigging that rely on UPS to boost strength, ones designed for operatives or light infantry.",
```
Extensive, protective, intrusive, wasn't 100% what word fit here the most.